app/v1/connectors/LiabilitiesConnector.scala

```diff
@@ -34,12 +36,14 @@ class LiabilitiesConnector @Inject()(val http: HttpClient,
 
   def retrieveLiabilities(request: LiabilitiesRequest)(implicit hc: HeaderCarrier,
                                                        ec: ExecutionContext,
-                                                       userRequest: UserRequest[_]): Future[DesOutcome[LiabilitiesResponse]] = {
+                                                       userRequest: UserRequest[_],
+                                                       correlationId: String): Future[DesOutcome[LiabilitiesResponse]] = {
 
     import v1.connectors.httpparsers.StandardDesHttpParser._
-    implicit val requestToDate: String = request.to
+    implicit val requestToDate: LocalDate = LocalDate.parse(request.to)
```

Comment:
instead of doing the parsing here, could it be done in the request parser after validation?

Reply:
this is to validate the response, not the request